Doc/faq/general.rst

```diff
@@ -222,7 +222,7 @@ releases are announced on the comp.lang.python and comp.lang.python.announce
 newsgroups and on the Python home page at https://www.python.org/; an RSS feed of
 news is available.
 
-You can also access the development version of Python through Mercurial.  See
+You can also access the development version of Python through Git.  See
 https://docs.python.org/devguide/faq.html for details.
 
```

Comment:
`https://docs.python.org/devguide` is still correct, but the lack of faq does need to be fixed.  Perhaps just remove the `/faq.html` part.